src/Scaffolding/VS.Web.CG.Mvc/Minimal Api/MinimalApiGenerator.cs

```diff
@@ -161,6 +161,7 @@ internal async Task AddEndpointsMethod(string membersBlockText, string endpoints
                     if (templateModel.OpenAPI)
                     {
                         usings.Add("Microsoft.AspNetCore.Http.HttpResults");
+                        usings.Add("Microsoft.AspNetCore.OpenApi");
                     }
```

Comment:
Note that this namespace is in a separate NuGet package (of the same name) that must be installed in the project. Does the scaffolder take care of ensuring the package it installed too?

Reply:
Yup VS will take care of that. On the cmd line, we want the dev to install it themselves. I am missing an error message if not present however, will add that for the cmd line scenario.

Reply:
done.